server/test/generateW2WebAPI.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+console.log("Generating WarpWorks WebAPI:");
+const warpCore = require('@warp-works/core');
+
+console.log("Reading SMN file...");
+var smn = warpCore.readFile(process.cwd()+"/../../mda/smnModels/WarpWorks.smn");
```

Comment:
I try to avoid using `process.cwd()` as much as possible as it really depends on where you start the program. I'm now using this logic in most places:

    const ROOT_DIR = path.dirname(require.resolve('./path/to/package.json'));

and then build on this value.